sql/core/src/test/scala/org/apache/spark/sql/StatisticsCollectionSuite.scala

```diff
@@ -168,6 +157,92 @@ class StatisticsCollectionSuite extends StatisticsCollectionTestBase with Shared
       assert(stats.simpleString == expectedString)
     }
   }
+
+  test("change stats after truncate command") {
+    val table = "change_stats_truncate_table"
+    Seq(false, true).foreach { autoUpdate =>
+      withSQLConf(SQLConf.AUTO_UPDATE_SIZE.key -> autoUpdate.toString) {
```

Comment:
`TruncateTableCommand` doesn't read the `AUTO_UPDATE_SIZE` config at all, we don't need to test with this config.